_docs_integrate/data-model-overview.md

```diff
@@ -70,31 +70,31 @@ A RelationshipTemplate serves two purposes:
+| id              | `string`                                                                                                                                         | {% include descr_id class="Relationship" prefix="REL" %}                                                                                                                                                                                                                                                                                                                                                                                                       |                                               |
+| template        | [`RelationshipTemplate`](#relationshiptemplate)                                                                                                  | The RelationshipTemplate that was used to establish this Relationship.                                                                                                                                                                                                                                                                                                                                                                                         |                                               |
+| status          | `"Pending"` \| `"Active"` \| `"Rejected"` \| `"Revoked"`                                                                                         | The status of this Relationship. <br>{::nomarkdown}<ul><li>Pending: the Relationship was initiated, but not yet accepted the recipient. In this state you cannot send Messages yet.</li><li>Active: this means that the Relationship is active. As long as it is active, both participants can exchange Messages.</li><li>Rejected: the Relationship was rejected by the recipient.</li><li>Revoked: the Relationship was revoked by the sender.</li></ul>{:/} |                                               |
+| creationContent | [`RelationshipCreationContent`](#relationshipcreationcontent) \| [`ArbitraryRelationshipCreationContent`](#arbitraryrelationshipcreationcontent) | The content sent along when the Relationship is initiated. If the other party uses the enmeshed App, `RelationshipCreationContent` has to be used. The `ArbitraryRelationshipCreationContent` can be filled with anything and serves as a fallback if the `RelationshipCreationContent` is not sufficient.                                                                                                                                                     | will be encrypted before sent to the Backbone |
```

Comment:
As I understand it, the `creationContent` is automatically filled with a RelationshipCreationContent if the Relationship was initiated after answering a Request contained within a RelationshipTemplateContent. The wording here suggests as if you have to actively use the RelationshipCreationContent. 

Furthermore, in this case the initiator of the Relationship would be the App user and the other party would not necessarily be an App user, as formulated here.